src/software/jetson_nano/services/motor.cpp

```diff
@@ -540,8 +540,8 @@ TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor
         TbotsProto::MotorControl_DirectVelocityControl direct_velocity =
             motor.direct_velocity_control();
         EuclideanSpace_t target_euclidean_velocity = {
-            -direct_velocity.velocity().y_component_meters(),
             direct_velocity.velocity().x_component_meters(),
+            direct_velocity.velocity().y_component_meters(),
```

Comment:
why is this change necessary? I suspect the original intentional is correct for coordinate transformation reasons.

right @nimazareian ?

Reply:
I'm going to revert this change bc I don't know what it does. Will need to test on robot